package.json

```diff
@@ -1888,6 +1900,14 @@
         "linux": "ctrl+right",
         "when": "calva:keybindingsEnabled && editorLangId == clojure && editorTextFocus && paredit:keyMap =~ /original|strict/ && config.calva.paredit.hijackVSCodeDefaults && !calva:cursorInComment || calva:cursorAfterComment"
       },
+      {
+        "command": "paredit.forwardSexpOrUp",
+        "when": "calva:keybindingsEnabled && editorLangId == clojure && editorTextFocus && paredit:keyMap =~ /original|strict/ && config.calva.paredit.hijackVSCodeDefaults && !calva:cursorInComment || calva:cursorAfterComment"
+      },
+      {
+        "command": "paredit.backwardSexpOrUp",
+        "when": "calva:keybindingsEnabled && editorLangId == clojure && editorTextFocus && paredit:keyMap =~ /original|strict/ && config.calva.paredit.hijackVSCodeDefaults && !calva:cursorInComment || calva:cursorAfterComment"
+      },
```

Comment:
I tested this now. The when expression does not automatically appear in the shortcut editor. So then it is probably better to just skip this.

Reply:
What do you mean by that? I see it in the command pallet if I search

Reply:
Yes, but this is key binding config, right? Just lacking the actual key. Unless I misread something. I was hoping that when you open the keyboard shortcuts editor, then the when context from this config would be filled in when you added a shortcut for one of these commands. But it doesn't. So then these configs are just noise.

Reply:
Ah, I misunderstood the purpose of this then. I thought it was required for plugin exports